services/src/main/java/org/keycloak/keys/DefaultKeyManager.java

```diff
@@ -75,7 +75,7 @@ public KeyWrapper getActiveKey(RealmModel realm, KeyUse use, String algorithm) {
             List<KeyProvider> providers = getProviders(realm);
             activeKey = getActiveKey(providers, realm, use, algorithm);
             if (activeKey != null) {
-                logger.infov("No keys found for realm={0} and algorithm={1} for use={2}. Generating keys.",
+                logger.warnv("No keys found for realm={0} and algorithm={1} for use={2}. Generating keys.",
```

Comment:
I don't think this should be a warn as it can be expected in some cases. For example if you switch algorithms.

Reply:
@samdammers That change is the only one that makes sense to keep from #7108. That is a fix from that PR.

Sorry for the re-work.

Reply:
All good, I will swap it back

Reply:
Thanks, running CI now.